zammad/values.yaml

```diff
@@ -397,6 +397,11 @@ autoWizard:
   #     ]
   #   }
 
+initJob:
```

Comment:
Can we move this to `.Values.zammadConfig.initJob`? There is already `initContainers` there, I think it would fit well.